README.rst

```diff
@@ -3,7 +3,7 @@
 .. image:: https://travis-ci.org/mandeep/cookiecutter-pyqt5.svg?branch=master
     :target: https://travis-ci.org/mandeep/cookiecutter-pyqt5
 
-Cookiecutter-PyQt5 is a Cookiecutter_ template that assists users in their creation of GUI applications.
+Cookiecutter-PyQt5 is a Cookiecutter_ template that assists users in their creation of PyQt5 GUI applications. All projects created from this cookiecutter will default to the GPLv3 license. For more details regarding Riverbank's license please visit: https://www.riverbankcomputing.com/commercial/license-faq
```

Comment:
"will default" suggests the user doesn't need to do anything extra here. For that to be the case, you'd need to either:

1. Tell the user they must update the license file for their own project to comply, or
2. Re-license this repo under GPL yourself (so the contents of the license file distributed with this repo would change).

I'm in favour of option 1 because it allows people with commercial licenses to also use your cookiecutter and won't cause hassles for anyone who has already forked this repo.

imo, it's not clear enough that the user needs to do something.